dataland-community-manager/src/main/kotlin/org/dataland/datalandcommunitymanager/model/dataRequest/DataRequestCompanyIdentifierType.kt

```diff
@@ -0,0 +1,17 @@
+package org.dataland.datalandbackend.model.enums.p2p
+
+import io.swagger.v3.oas.annotations.media.Schema
+
+/**
+ * An enum for the different identifier types that an identifier value of a stored data request can have.
+ */
+@Schema(
+    enumAsRef = true,
+)
+enum class DataRequestCompanyIdentifierType {
+    Lei,
+    Isin,
+    PermId,
+    MultipleRegexMatches,
```

Comment:
What exactly is this identifier?

Reply:
This enum contains all possible outcomes of the regex-matching of the company identifier
PLUS
the special case DatalandCompanyId.

If a company identifier matches several types (e.g. LEI and PErmID), the matching gets the result "MultipleRegexMatches".

If a company identifier can be cross-referenced with a company that already exists in our backend, the type switches to "DatalandCompanyId".

Reply:
Ok, thank you